### PR TITLE
Docs: Add RisingWave

### PR DESCRIPTION
**Pull request by hengm3467**
Adds an overview page about RisingWave's support for Iceberg to help developers get started with the RisingWave Iceberg integration.

### CODE REVIEW

**Review by nastra — APPROVED** [review]

**Review by jbonofre — APPROVED** [review]

**Comment by hengm3467** [comment]
@Fokko Could you please take a look? Thanks! And let me know if you have any questions.

File: docs/docs/risingwave.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+## Table Formats and Warehouse Locations
+
+Currently, RisingWave only supports the Iceberg V2 table format and S3-compatible object storage as Iceberg warehouse locations.
+
+## Catalogs
+
+RisingWave supports the following catalogs:
+
+- `rest`
+- `jdbc`
```
Comment (Fokko) [inline docs/docs/risingwave.md:38]: This is only called JDBC in Java, in PyIceberg and Iceberg-Rust it is called the `sql` catalog. This reminds us that naming things is one of the hardest things in computer science.

```suggestion
- `jdbc` / `sql`
```

Considering that Risingwave is Postgres compatible, you could also use the SQL Catalog backed by Risingwave itself :)

File: docs/docs/risingwave.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+ - limitations under the License.
+ -->
+
+# RisingWave
+
+[RisingWave](https://risingwave.com/) is a Postgres-compatible SQL database designed for real-time event streaming data processing, analysis, and management. It can ingest millions of events per second, continuously join and analyze live data streams with historical tables, serve ad-hoc queries in real-time, and deliver fresh, consistent results.
+
+## Supported Features
+
+RisingWave supports batch reads and streaming writes of Iceberg tables via its built-in source and sink connectors. For more information, see the [Iceberg source connector documentation](https://docs.risingwave.com/integrations/sources/apache-iceberg) and [Iceberg sink connector documentation](https://docs.risingwave.com/integrations/destinations/apache-iceberg).
```
Comment (Fokko) [inline docs/docs/risingwave.md:27]: Can we add the full name once, including the trademark?
```suggestion
RisingWave supports batch reads and streaming writes of Apache Iceberg™ tables via its built-in source and sink connectors. For more information, see the [Iceberg source connector documentation](https://docs.risingwave.com/integrations/sources/apache-iceberg) and [Iceberg sink connector documentation](https://docs.risingwave.com/integrations/destinations/apache-iceberg).
```

**Review by Fokko — APPROVED** [review]
@hengm3467 Sorry for the late reply, and thanks for pinging me. I left two nits, apart from that it looks good to me 👍

File: docs/docs/risingwave.md (hunk 1 above)
Comment (hengm3467) [reply docs/docs/risingwave.md]: > This is only called JDBC in Java, in PyIceberg and Iceberg-Rust it is called the `sql` catalog. This reminds us that naming things is one of the hardest things in computer science.
> 
> Considering that Risingwave is Postgres compatible, you could also use the SQL Catalog backed by Risingwave itself :)

I echoed the point that naming things is one of the hardest things. :) 
Much appreciate your suggested change.

**Comment by hengm3467** [comment]
@Fokko No worries. And thank you so much for the review! :) Have a nice day.

**Pull request merged**